### PR TITLE
Test logging to file descriptor

### PR DESCRIPTION
**Pull request by plu**

### CODE REVIEW

**Comment by plu** [comment]
The test will fail on Mac OS Sierra. I still have to figure out why.

**Comment by lawrencelomax** [comment]
> The test will fail on Mac OS Sierra. I still have to figure out why

This could be related to the new `logd` API. We could extract away the usage of `ASL` for macOS versions that have it. It would require another implementation, but that should be simple enough.

**Comment by lawrencelomax** [comment]
lgtm!

**Comment by lawrencelomax** [comment]
@plu funtimes with `.pbxproj` merging. Could you merge/rebase and put up again?

**Comment by plu** [comment]
Done.

**Comment by plu** [comment]
I had a quick glance at the new `logd` API, looks like they do not support a `logd` version of `asl_add_output_file`.

**Comment by lawrencelomax** [comment]
I had some merge issues with this PR, so I'll need to rebase it.

File: FBControlCoreTests/Tests/FBControlCoreLoggerTests.m [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "FBControlCoreLogger.h"
```
Comment (lawrencelomax) [inline FBControlCoreTests/Tests/FBControlCoreLoggerTests.m]: Can you make this a Framework Import of `FBControlCore` to keep in line with the rest of the tests:

```
#import <FBControlCore/FBControlCore.h>
```

**Comment by plu** [comment]
This is still incomplete, it only contains a test case. There is a difference between El Capitan and Sierra. The test case makes this difference visible: On El Capitan it passes, on Sierra it fails.

**Comment by plu** [comment]
Since this is still incomplete I will close it for now and come back later with another pull request for this topic :).

**Pull request closed without merging**